Conformity/custom-rule/custom-rule.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/local/bin/python3
+
+from genericpath import isdir
```

Comment:
To make sure that we have all the dependencies satisfied, we could provide a `requirements.txt` :)